templates/bin_setup_review_app.erb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+# Run this script to set up a review app's database and worker dyno
+
+set -e
+
+if [ -z "$1" ]; then
+  printf "You must provide a review app (same as the pull request) id.\n"
+  exit 64
+else
```

Comment:
We can reduce indentation by taking advantage of the fact that `exit` terminates the program.

```sh
if [ -z "$1" ]; then
  echo "You must provide a review app (same as the pull request) id."
  exit 64
heroku # ...